### DISCUSSION TITLE
What happened to the README?

### DISCUSSION THREAD

**Discussion by Lietsaki** [pmndrs/drei#2091] [Q&A]
The readme has changed from showing all components to just linking to https://drei.docs.pmnd.rs, where you need to check components **one by one** from the sidebar.

You can no longer find a component or get inspiration at a glance on the repo's homepage. This is incredibly cumbersome and counter-intuitive.

Is there a way to see all components in the same page as before? I would strongly suggest bringing back the previous readme layout.

**Comment by nady90** [comment]
The readme used to be way easier to browse through. I agree we need the old readme back.
